integrations/symfony-full-configuration.rst

```diff
@@ -78,6 +78,7 @@ This page shows an example of all configuration values provided by the bundle.
                 service: 'my_service'       # Can not be used with "factory" or "config"
                 flexible_client: false      # Can only be true if http_methods_client is false
                 http_methods_client: false  # Can only be true if flexible_client is false
+                public: false               # Set to true if you really cannot use dependency injection and need to make the client service public
```

Comment:
What to do with this. Default is `null`.

Reply:
@dbu Should I change it ? Or is it fine like this.

Reply:
ups, i was too quick. lets say `public: null` and keep the rest as is?

Reply:
I will change it!

Reply:
@dbu https://github.com/php-http/documentation/pull/242